From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: DigitalRegent <misterwener@gmail.com>
Date: Sat, 11 Apr 2020 13:10:58 +0200
Subject: [PATCH] Brand support

TODO(Proximyst): Fixup this into the other brand support patch

diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
index 105572b2847207e83580808bba3b5abf6960d049..31aad6440e23ef3bbef90418923747a71709854c 100644
--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
@@ -2381,7 +2381,7 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
     // Paper start - brand support
     @Override
     public String getClientBrandName() {
-        return getHandle().playerConnection.getClientBrandName();
+        return getHandle().playerConnection != null ? getHandle().playerConnection.getClientBrandName() : null;
     }
     // Paper end
 
